### PR TITLE
Add webchat

### PR DESCRIPTION
**Pull request by lucasefe**
* Adds an option to the broadly settings page to enable the webchat. 
* If enabled, it injects the webchat script. 
* Adds a DEVELOPMENT.md and a docker-compose.yml files to launch an instance of wordpress without messing with your local env. 

https://github.com/broadly/broadly/issues/2597

### CODE REVIEW

File: broadly.php [hunk 1]
```diff
@@ -22,7 +22,7 @@
 	 */
 	class Broadly_Plugin {
 
-		public static $version = '2.0.3';
+		public static $version = '2.1.0';
```
Comment (assaf) [inline broadly.php]: Let's release this as 3.0.0 to match with the app

**Comment by lucasefe** [comment]
Should I merge? Not sure how to publish it either.

**Comment by assaf** [comment]
I'm going to try and figure out this SVN thing, got a minute to do a quick review, need second pair of eyes on: https://github.com/broadly/wordpress-plugin/pull/11 and https://github.com/broadly/wordpress-plugin/pull/10

**Comment by lucasefe** [comment]
On https://github.com/broadly/wordpress-plugin/pull/10, besides adding webchat, which is the main feature, we are also serving reviews from https, and improved the settings page to inform about how to send contact form info to Broadly Engage, but maybe you don't want to say that there.

**Comment by assaf** [comment]
Yeah, tried to keep it short and simple so only focused on Web Chat

**Comment by assaf** [comment]
I'm going to try figure out SVN now, Famous Last Words(tm)

**Pull request merged**